### DISCUSSION TITLE
Backwards compatibility

### DISCUSSION THREAD

**Discussion by skyetomez** [open-ephys/plugin-GUI#552] [Q&A]
Hello, we have data recorded in openephys from version 0.4.4.1 but the current gui neither recognizes these directories nor their data files. Is there a way to analyze these files in the current version of open-ephys? 

Here is the header from the `settings.xml`

```xml

<?xml version="1.0" encoding="UTF-8"?>
   
   <SETTINGS>
     <INFO>
       <VERSION>0.4.4.1</VERSION>
       <PLUGIN_API_VERSION>5</PLUGIN_API_VERSION>
       <DATE>19 Jun 2019 14:17:39</DATE>
       <OS>Windows 8.0</OS>
       <MACHINE>NEURON-JEREMY</MACHINE>
```

I can add more information if needed. We're currently using version 6.3. of the gui

**Comment by oepavel** [comment]
Hi Skyler,  
If you can send a zip or link to an example of recorded data we can send back a script that will convert the data for you.  
Best, 
Pavel 
pavel@open-ephys.org

**Reply by skyetomez** [reply]
I will try. I'm downloading a zip now but it's already 10GB of data. I don't know the upload limit of zip files. 
Is there another format I can do?

Perhaps share a dropbox/google doc link with my edu email?

**Reply by medengineer** [reply]
Can you send me an e-mail at pavel@open-ephys.org with the data format used and we can go from there. Thanks!

**Reply by skyetomez** [reply]
Is the data format specified in the `settings.xml` or where is this found?

**Reply by medengineer** [reply]
Yes, can you paste the entire output of the settings.xml file here.
